library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/api/BlockContentRegistries.java

```diff
@@ -105,6 +105,7 @@ public class BlockContentRegistries {
 
 						return DataResult.success(block);
 					}))
+			.filter(block -> block.getDefaultState().contains(Properties.AXIS))
```

Comment:
Just noticed this.

While I like the idea, I also find it a bit bad, thought that comes from strippable being kind of awful from the start.
Allowing other kind of blocks to be strippable would be interesting but that's a debate for another PR. But it shows that the conditions system has potential.